docker-compose.yml

```diff
@@ -1,4 +1,5 @@
-version: '3.4'
+# the attribute `version` is obsolete
+# version: '3.4'
```

Comment:
```suggestion
```
Should delete it instead of commenting it out

Reply:
thanks, done!